masterlist.yaml

```diff
@@ -4767,6 +4767,25 @@ plugins:
       # ELFX
       - crc: 0xAD111166
         util: 'FO4Edit v4.1.5k'
+
+  - name: 'Rebuild_Settlements.esp'
+    url: [ 'https://www.nexusmods.com/fallout4/mods/79618' ]
+    dirty:
+      - <<: *quickClean
+        crc: 0x7CA58943
+        util: '[FO4Edit v4.1.5f](https://www.nexusmods.com/fallout4/mods/2737)'
+        itm: 3
+    clean:
+      - crc: 0x7242E237
+        util: 'FO4Edit v4.1.5f'
+  - name: 'Rebuild_Settlements_Previs.esp'
+    url: [ 'https://www.nexusmods.com/fallout4/mods/79618' ]
```

Comment:
```suggestion
    url: [ 'https://www.nexusmods.com/fallout4/mods/79618/' ]
```

Reply:
Fixed in 22880362f1ab48beb6616916ade672ce643ea8c9